src/baseclasses/chemical_energy/potentiometry_gaschromatography.py

```diff
@@ -326,16 +350,34 @@ class NECCPotentiostatMeasurement(ArchiveSection):
     minimum_working_electrode_potential = Quantity(type=np.dtype(np.float64), unit='V')
     maximum_working_electrode_potential = Quantity(type=np.dtype(np.float64), unit='V')
 
+    mean_counter_electrode_potential = Quantity(type=np.dtype(np.float64), unit='V')
+    standard_deviation_counter_electrode_potential = Quantity(type=np.dtype(np.float64), unit='V')
+    minimum_counter_electrode_potential = Quantity(type=np.dtype(np.float64), unit='V')
+    maximum_counter_electrode_potential = Quantity(type=np.dtype(np.float64), unit='V')
+
+    mean_ewe_ece_difference = Quantity(type=np.dtype(np.float64), unit='V')
+    standard_deviation_ewe_ece_difference = Quantity(type=np.dtype(np.float64), unit='V')
+    minimum_ewe_ece_difference = Quantity(type=np.dtype(np.float64), unit='V')
+    maximum_ewe_ece_difference = Quantity(type=np.dtype(np.float64), unit='V')
+
+    def calculate_statistics(self, quantity, quantity_name):
+        supported_quantities = {'current', 'working_electrode_potential', 'counter_electrode_potential', 'ewe_ece_difference'}
+        if quantity_name not in supported_quantities:
+            return
+        if quantity is not None:
```

Comment:
```py
if quantity is None:
   return
...
```

Reply:
not decisive here, but we should always practice, first writing guard statements and then setting stuff (in also dont always do it i know :))

Reply:
Resolved :white_check_mark: